From 03197667856d970ff253756fe611756ad4eecce3 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Wed, 20 Sep 2023 12:50:04 +0200
Subject: [PATCH 0583/1141] arm64: dts: st: remove goodix pinmux on
 stm32mp25-pinctrl.dtsi

Goodix is no more supported on mp25 boards.

Change-Id: Ie77468a5f88db06338dd10aff60ed801ffcecc42
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/329412
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp25-pinctrl.dtsi | 7 -------
 1 file changed, 7 deletions(-)

--- a/arch/arm64/boot/dts/st/stm32mp25-pinctrl.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp25-pinctrl.dtsi
@@ -136,13 +136,6 @@
 		};
 	};
 
-	goodix_pins_a: goodix-0 {
-		pins {
-			pinmux = <STM32_PINMUX('I', 13, GPIO)>;
-			bias-pull-down;
-		};
-	};
-
 	i2c2_pins_a: i2c2-0 {
 		pins {
 			pinmux = <STM32_PINMUX('B', 5, AF9)>, /* I2C2_SCL */
